activities/VideoViewer.activity/lib/tutorial.js

```diff
@@ -0,0 +1,86 @@
+define(["webL10n"], function (l10n) {
+	var tutorial = {};
+
+	tutorial.start = function() {
+		var steps = [
```

Comment:
Your file mixed indentation with TAB and identation with white spaces. Could you use one sort of indentation only?